std/os/index.zig

```diff
@@ -39,6 +39,12 @@ pub const File = @import("file.zig").File;
 pub const time = @import("time.zig");
 
 pub const page_size = 4 * 1024;
+pub const PATH_MAX = switch (builtin.os) {
+    Os.linux => linux.PATH_MAX,
```

Comment:
Is this derived using [pathconf](http://man7.org/linux/man-pages/man3/pathconf.3.html)?